gitlab/Makefile

```diff
@@ -0,0 +1,9 @@
+ROOT_DIR = ..
+include ../_scripts/Makefile.projects
+
+.PHONY: config # Configure .env file
+config:
```

Comment:
this should be `config-hook`. `config` is an internal target that calls your `config-hook`